schema/server/branch.json

```
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "version": "0.0.1",
+  "type": "object",
+  "properties": {
+    "name": {
+      "oneOf": [
+        { "type": "null" },
+        { "type": "string" }
+      ]
```

```suggestion
      "type": ["null", "string"]
```

does this more compactly (& elsewhere)

---

done yh much better